Core/HLE/sceKernelInterrupt.cpp

```diff
@@ -752,6 +752,17 @@ static u32 sysclib_memset(u32 destAddr, int data, int size) {
 	return 0;
 }
 
+static int sysclib_strstr(u32 s1, u32 s2) {
+	ERROR_LOG(SCEKERNEL, "Untested sysclib_strstr(%08x, %08x)", s1, s2);
+	std::string str1 = Memory::GetCharPointer(s1);
+	std::string str2 = Memory::GetCharPointer(s2);
+	int index = str1.find(str2);
+	if (index < 0) {
```

Comment:
I'd suggest, to avoid warnings:

```c++
size_t index = str1.find(str2);
if (index == str1.npos) {
    return 0;
}
return s1 + (uint32_t)index;
```

Although, it ought to be the same.

-[Unknown]